WooCommerce/Classes/Authentication/Navigation Exceptions/ApplicationPasswordDisabledViewModel.swift

```diff
@@ -37,7 +38,10 @@ struct ApplicationPasswordDisabledViewModel: ULErrorViewModel {
     }
 
     func didTapPrimaryButton(in viewController: UIViewController?) {
-        // no-op
+        guard let viewController else {
+            return
+        }
+        WordPressAuthenticator.showLoginForJustWPCom(from: viewController)
```

Comment:
I was wondering whether there is a way to unit test this call to a static method. I also noticed that we don't have unit tests for this `ApplicationPasswordDisabledViewModel`.

Reply:
I added tests in c2145a7dc6a730bf71a2e565f27c4c53ad23291c and 835043d63790eb4b114135badd62675d5f50b7f0. 

> I was wondering whether there is a way to unit test this call to a static method.

I recall having written a mock for `WordPressAuthenticator` and inject the mock through the initializer to check the trigger of static methods. In this case, I'm only checking for the type of the presented controller, as that should be enough.